evil-collection-magit-todos.el

```diff
@@ -0,0 +1,46 @@
+;;; evil-collection-magit-todos.el --- Bindings for `magit-todos' -*- lexical-binding: t -*-
+
+;; Copyright (C) 2018 James Nguyen
```

Comment:
@fbergroth: Your name should go here.

Reply:
Ah right, thanks. I don't care to update it though :)